### PR TITLE
Irmprove wording, fix example that returns ServiceController

### PR DESCRIPTION
**Pull request by rkeithhill**
Version(s) of document impacted
------------------------------
- [x] Impacts 7 document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

File: reference/6/Microsoft.PowerShell.Core/Register-ArgumentCompleter.md [hunk 1]
```diff
@@ -193,7 +193,7 @@ Accept wildcard characters: False
 ### -ScriptBlock
 
 Specifies the commands to run to perform tab completion. The script block you provide should return
-the values complete the input. The script block must unroll the values using the pipeline
+the values to complete the input. The script block must unroll the values using the pipeline
```
Comment (rkeithhill) [inline reference/6/Microsoft.PowerShell.Core/Register-ArgumentCompleter.md]: I'm not 100% on the `to` here but without it, the sentence doesn't read right IMO.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[e5fb226](https://github.com/rkeithhill/PowerShell-Docs/pull/4339/commits/e5fb2265aec29523c60f1638551bb8a8f20bf117)_: 

:clock10: **Full build**: the average full build time is _13 min(s) 23 sec(s)_, this is based on the last _41_ build(s) for this repository.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[e5fb226](https://github.com/rkeithhill/PowerShell-Docs/pull/4339/commits/e5fb2265aec29523c60f1638551bb8a8f20bf117)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Core/Register-ArgumentCompleter.md](https://github.com/rkeithhill/PowerShell-Docs/blob/fix-register-argumentcompleter/reference/6/Microsoft.PowerShell.Core/Register-ArgumentCompleter.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/register-argumentcompleter?view=powershell-6&branch=pr-en-us-4339) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C23%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201905230524410259-4339%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=LKcZj9xzNEFPNhxa0Q8RX5rTJtYQfqMmS5u2KPFzYj4%3D&st=2019-05-23T05%3A34%3A45Z&se=2019-06-23T05%3A39%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[fdc3f18](https://github.com/rkeithhill/PowerShell-Docs/pull/4339/commits/fdc3f180c953d091157790deb57599286d1fc935)_: 

:clock10: **Full build**: the average full build time is _13 min(s) 20 sec(s)_, this is based on the last _44_ build(s) for this repository.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[fdc3f18](https://github.com/rkeithhill/PowerShell-Docs/pull/4339/commits/fdc3f180c953d091157790deb57599286d1fc935)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Core/Register-ArgumentCompleter.md](https://github.com/rkeithhill/PowerShell-Docs/blob/fix-register-argumentcompleter/reference/6/Microsoft.PowerShell.Core/Register-ArgumentCompleter.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/register-argumentcompleter?view=powershell-6&branch=pr-en-us-4339) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C23%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201905231448204167-4339%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Yn5Z0R4693GfQqnKnf%2BY8k8gtq9vgT%2B%2BYQkNV8Xwet8%3D&st=2019-05-23T14%3A57%3A41Z&se=2019-06-23T15%3A02%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/6/Microsoft.PowerShell.Core/Register-ArgumentCompleter.md [hunk 2]
```diff
@@ -193,11 +193,11 @@ Accept wildcard characters: False
 ### -ScriptBlock
 
 Specifies the commands to run to perform tab completion. The script block you provide should return
-the values to complete the input. The script block must unroll the values using the pipeline
+the values that complete the input. The script block must unroll the values using the pipeline
```
Comment (rkeithhill) [inline reference/6/Microsoft.PowerShell.Core/Register-ArgumentCompleter.md]: Thanks!

File: reference/6/Microsoft.PowerShell.Core/Register-ArgumentCompleter.md [hunk 3]
```diff
@@ -193,11 +193,11 @@ Accept wildcard characters: False
...
 (`ForEach-Object`, `Where-Object`, etc.), or another suitable method. Returning an array of values
 causes PowerShell to treat the entire array as **one** tab completion value.
 
-The script block should also accept the following parameters in the order specified below. The names
+The script block must accept the following parameters in the order specified below. The names
```
Comment (rkeithhill) [inline reference/6/Microsoft.PowerShell.Core/Register-ArgumentCompleter.md]: 👍

**Pull request merged**